include/db_connector

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Prowler - the handy cloud security tool (copyright 2018) by Toni de la Fuente
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+# DB connectors
+# Supported connectors list is stored in SUPPORTED_DB_PROVIDERS
+# Once the logic of the connector is added it is needed to add it to the list also
+# Example: adding oracle to the list
+# SUPPORTED_DB_PROVIDERS="postgresql|oracle"
+# oracle_connector() { ... }
+
+SUPPORTED_DB_PROVIDERS="postgresql"
+
+# Connectors for different databases
+#POSTGRESQL variables
+PSQL_USER=$(awk -F ':' '{print $4}' ~/.pgpass)
+PSQL_HOSTNAME=$(awk -F ':' '{print $1}' ~/.pgpass)
+PSQL_PORT=$(awk -F ':' '{print $2}' ~/.pgpass)
+PSQL_TABLE=$(awk -F ':' '{print $6}' ~/.pgpass)
+
```

Comment:
We need to handle the error if the user has not permissions to open the file or it does not exist:
```
awk: can't open file /Users/user/.pgpass
 source line number 1
```

Reply:
Good catch @sergargar !!